src/Components/LifecycleChart/LifecycleChart.tsx

```diff
@@ -75,6 +132,13 @@ const LifecycleChart: React.FC<LifecycleChartProps> = ({ lifecycleData }: Lifecy
     );
   };
 
+  const fetchTicks = () => {
+    const a = updatedLifecycleData.map((data) => {
```

Comment:
I made these bad variable names while I was debugging. I take responsibility for that. But I don't want them in the code. :) 
```suggestion
    return updatedLifecycleData.map((data) => {
```